spec/models/topic_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+	scenario 'can a topic be saved' do
```

Comment:
`scenario` is feature language. In unit tests we use `it`. It reads better. 
```
it 'can a topic be saved' do
```
We only use scenario in feature specs (aka system specs), because features specs have more than one expectation which is hard to sum up in a single "it should do this" type of way.